README.md

```diff
@@ -139,6 +139,8 @@ The following [dispatchers](./telemetry/producer.go#L10-L19) are supported
   * Override stream names with env variables: KINESIS_STREAM_\*uppercase topic\* ex.: `KINESIS_STREAM_V`
 * Google pubsub: Along with the required pubsub config (See ./test/integration/config.json for example), be sure to set the environment variable `GOOGLE_APPLICATION_CREDENTIALS`
 * Logger: This is a simple STDOUT logger that serializes the protos to json.
+  
+>NOTE: If you want to add a new dispatcher, it must have an integration test and updated documentation associated with it.
```

Comment:
NOTE: To add a new dispatcher, please provide integration tests and updated documentation.

Reply:
updated